src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -811,6 +811,19 @@ public static void DragCoordinates(this IApp app, float fromX, float fromY, floa
 			});
 		}
 
+		/// <summary>
+		/// Performs a pan gesture between 2 points.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="fromX">The x coordinate to start panning from.</param>
+		/// <param name="fromY">The y coordinate to start panning from.</param>
+		/// <param name="toX">The x coordinate to pan to.</param>
+		/// <param name="toY">The y coordinate to pan to.</param>
+		public static void Pan(this IApp app, float fromX, float fromY, float toX, float toY)
+		{
+			app.DragCoordinates(fromX, fromY, toX, toY);
```

Comment:
Similar to https://github.com/xamarin/Xamarin.Forms/blob/2f8f4864a4d289dc89a6228e2ca9d6a49993e365/Xamarin.Forms.Core.UITests.Shared/Utilities/Gestures.cs#L88 in Xamarin.Forms.

Allow to port Xamarin.UITests more easily to .NET MAUI.